app/javascript/article-form/articleForm.jsx

```diff
@@ -18,18 +21,6 @@ import Errors from './elements/errors';
 import KeyboardShortcutsHandler from './elements/keyboardShortcutsHandler';
 import Tags from '../shared/components/tags';
 
-const setupImageButton = ({ className = '', imgSrc, imgAltText = '', onClickCallback }) => {
```

Comment:
is there a reason why this refactoring was undone? It was merged literally 6 days ago eheh https://github.com/thepracticaldev/dev.to/pull/4159